tests/test_unit.py

```diff
@@ -44,6 +44,15 @@ def test_copy():
     assert slug == ureg.slug
 
 
+def test_compatibility():
+    ureg = ansunits.UnitRegistry()
+
+    for each in (ureg.ft, ureg.N, ureg.delta_K):
+        unit_comp = each.compatible_units()
+        for unit in unit_comp:
+            assert each.dimensions == ansunits.Unit(unit).dimensions
```

Comment:
@ansaboutin please can you make the test more explicit? I.e. check explicitly that particular units are and are not included. E.g., you including a temperature difference here, and that's a good example where I believe we should very explicitly check what is and is not in that list. We can also look at torque and energy units. The most important aspect of that is that the test will very clearly show what levels of compatibility we are actually accepting here.